classes/04class/exercises/c04-iac02/nimmi89/variables.tf

```diff
@@ -0,0 +1,37 @@
+#Declare the input variables
+
+variable "vpcCIDRblock" {
+  type        = string
+  description = "CIDR block for the VPC"
+}
+
+variable "public_subnet_cidr_blocks" {
+  type        = list
+  description = "List of public subnet CIDR blocks"
+}
+
+variable "public_subnet_names" {
+  type        = list
+  description = "List of public subnets names"
+}
+
+variable "private_subnet_cidr_blocks" {
+  type        = list
+  description = "List of private subnet CIDR blocks"
+}
+
+variable "private_subnet_names" {
+  type        = list
+  description = "List of private subnets names"
+}
+
```

Comment:
it is possible to group this input in a more complex data structure using list, map and object. So you'd have a more generic and flexible input